src/utils/thread_helper.cpp

```diff
@@ -1328,6 +1328,12 @@ otError ThreadHelper::RetrieveTelemetryData(Mdns::Publisher *aPublisher, threadn
             srpServerResponseCounters->set_name_exists_count(responseCounters->mNameExists);
             srpServerResponseCounters->set_refused_count(responseCounters->mRefused);
             srpServerResponseCounters->set_other_count(responseCounters->mOther);
+#if OTBR_DNS_UPSTREAM_QUERY
```

Comment:
I think the naming convention is `OTBR_ENABLE_<FEATURE>`.

E.g. https://github.com/openthread/ot-br-posix/blob/f432e672abe43803ef66f4677ce1a4b5645b824d/src/border_agent/border_agent.cpp#L251

Reply:
Thanks!!